src/aggregator/aggregator/generic_elem.go

```diff
@@ -106,14 +106,14 @@ type typeSpecificElemBase interface {
 }
 
 type lockedAggregation struct {
-	aggregation typeSpecificAggregation
-	sourcesSeen map[uint32]*bitset.BitSet
-	mtx         sync.Mutex
-	dirty       bool
+	aggregation   typeSpecificAggregation
+	sourcesSeen   map[uint32]*bitset.BitSet
+	mtx           sync.Mutex
+	lastUpdatedAt xtime.UnixNano
```

Comment:
this is just to compact the struct to be smaller?

Reply:
yep, this saves 4-8 bytes of padding